sdk/communication/AzureCommunication/AzureCommunication.xcodeproj/project.pbxproj

```diff
@@ -131,9 +131,9 @@
 			isa = PBXGroup;
 			children = (
 				0A3A5EC82316DB1E00473FDA /* Info.plist */,
-				1D07222C24C8FB0F00C2EF4E /* CommunicationUserCredentialTests.swift */,
-				8856CEE4253E3AEF00044559 /* ObjCCommunciationUserCredentialTests.m */,
-				88CC8C34254750260028977C /* ObjCCommunicationUserCredentialAsyncTests.m */,
+				1D07222C24C8FB0F00C2EF4E /* CommunicationTokenCredentialTests.swift */,
+				8856CEE4253E3AEF00044559 /* ObjCCommunciationTokenCredentialTests.m */,
```

Comment:
Typo: ObjC**Communciation**TokenCredentialTests

Reply:
This should have been renamed